views/base.html

```diff
@@ -174,6 +185,24 @@
     </footer>
     <!-- end of footer -->
 
+    <script>
+      function sendEvent(format, resourceName) {
```

Comment:
You have `sendEvent` function in both `base.html` and `showcase.html`. Remember that showcase.html extends base.html

Reply:
I guess this function is outdated and should be removed?